R/mechanism-gaussian.R

```diff
@@ -19,18 +19,11 @@ mechanismGaussian$methods(
 mechanismGaussian$methods(
     evaluate = function(fun, x, sens, postFun) {
         x <- censordata(x, .self$var.type, .self$rng)
-        if (.self$var.type %in% c('numeric', 'integer', 'logical')) {
-            if (NCOL(x) > 1) {
-                x <- fillMissing2d(x, .self$var.type, .self$impute.rng)
-            } else {
-                x <- fillMissing(x, .self$var.type, .self$impute.rng[1], .self$impute.rng[2])
-            }
-        } else {
-            x <- fillMissing(x, .self$var.type, categories=.self$bins)
-        }
+        x <- fillMissing(x, .self$var.type, impute.rng=.self$rng, categories=.self$bins)
         field.vals <- .self$getFunArgs(fun)
         true.val <- do.call(fun, c(list(x=x), field.vals))
         scale <- sens * sqrt(2 * log(1.25 / .self$delta)) / .self$epsilon
+        noise <- dpNoise(n=length(true.val), scale=scale, dist='gaussian')
```

Comment:
Why is this here?

Reply:
I must have added that for testing purposes at some point. It should not be there.